packages/server-common/src/handlers/webhooks/events-updated-process.handler.ts

```diff
@@ -40,6 +40,11 @@ export const webhookEventUpdatedProcessHandlerFactory = (
       const resourceId = record.messageAttributes.ResourceId?.stringValue;
       const channelId = record.messageAttributes.ChannelId?.stringValue;
 
+      if (!resourceId) {
+        logger.debug('Empty resourceId');
```

Comment:
ok first of all i think there is a check just underneath that sort of handles this already, I would probably refactor it if we want to bail out without errors for when `resourceId` is missing

second - how are we ever receiving payload from google with an empty resourceId? I would have thought this would handle it https://github.com/yldio/asap-hub/blob/95be5f71aabfe7b1eb0eaeb9bd76817203de57b7/packages/server-common/src/handlers/webhooks/events-updated.handler.ts#L42-L49

finally - why do you think that upon receiving an empty resourceId we could log and return? To me that's a legitimate error, I think we want to throw here?